LICENSE

```diff
@@ -50,6 +50,11 @@ authors.
   - http://apache.mirrors.ovh.net/ftp.apache.org/dist/xerces/j/source/
   - [Apache Software License version 2.0][Apache-2.0]
 
+* juniversalchardet
+  - Java port of universalchardet from Mozilla
+  - https://code.google.com/archive/p/juniversalchardet/source/default/source
```

Comment:
The author of the repository is using the username 'takscape', should we write that in the file ?
Since this is a port of a library from the Mozilla Foundation, we might as well need to include it too.